kubernetes/architecture.json

```diff
@@ -3,9 +3,6 @@
   "unreleased": true,
   "type": "architectural",
   "displayName": "Architecture",
-  "requiredProperties": [
```

Comment:
Is `requiredProperties: ['host']` just a default somewhere or was this not needed?